packages/DropdownMenu/stories/examples/Showcase.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Rule, Tagline } from "storybook/assets/styles/common.styles";
+import Heading from "@paprika/heading";
+import { DropdownMenuStory } from "../DropdownMenu.stories.styles";
+import DropdownMenuExample from "./DropdownMenuExample";
+
+const ExampleStory = () => (
+  <DropdownMenuStory>
+    <Heading level={1} displayLevel={2} isLight>
+      Showcase
+    </Heading>
+    <Tagline>Use the knobs to tinker with the props.</Tagline>
```

Comment:
Ha. This is still "experimental".  Though I thought it might be fun if each developer replaced it with their own joke... like:
https://github.com/acl-services/paprika/blob/master/packages/Sortable/stories/examples/Showcase.js#L30